### PR TITLE
Add ability to override redirectUrl in swagger-oauth

### LINKED ISSUE #457: Ability to override redirectUrl in swagger-oauth
I found hardcoded link in swagger-oauth 

var redirectUrl = host.protocol + '//' + host.host + "/o2c.html";

Could you move this to settings?

### PR DESCRIPTION
**Pull request by mohsen1**
Fixes #457

### CODE REVIEW

**Comment by fehguy** [comment]
@mohsen1 looks good.  Will this change the HTML structure for existing deployments?  Do we need to document anything for the upgrade?

File: README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ onComplete | This is a callback function parameter which can be passed to be not
 onFailure | This is a callback function parameter which can be passed to be notified of when SwaggerUI encountered a failure was unable to render.
 highlightSizeThreshold | Any size response below this threshold will be highlighted syntactically, attempting to highlight large responses can lead to browser hangs, not including a threshold will default to highlight all returned responses.
 supportedSubmitMethods | An array of of the HTTP operations that will have the 'Try it out!` option. An empty array disables all operations. This does not filter the operations from the display.
+redirectUrl | OAuth redirect URL
```
Comment (mohsen1) [inline README.md]: @fehguy no you just pass the `redirectUrl` property to `SwaggerUi` constructor

**Comment by fehguy** [comment]
@mohsen1 suggest you use a different name for the redirectUrl.  Perhaps `oauth2RedirectUrl`

**Comment by mohsen1** [comment]
@fehguy done

**Pull request merged**